### PR TITLE
fix: fix spacing for multi args

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

see: https://github.com/mongodb/mongocli/pull/833/files#r707760655

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: cobra2snooty_test.go [hunk 1]
```diff
@@ -57,6 +57,10 @@ func Echo() *cobra.Command {
 		Short:   "Echo anything to the screen",
 		Long:    "an utterly useless command for testing",
 		Example: "Just run root echo",
+		Annotations: map[string]string{
+			"args":                "string to print",
+			"string to printDesc": "A string to print",
+		},
```
Comment (melissamahoney-mongodb) [inline cobra2snooty_test.go:63]: Should the test include multiple and required args?
```suggestion
		Annotations: map[string]string{
			"args":                "string to print, second string",
			"requiredArgs":  "string to print, second string",
			"string to printDesc": "A string to print",
			"second stringDesc": "A second string",
		},
```

**Review by melissamahoney-mongodb — APPROVED** [review]
One nit, up to you, otherwise LGTM

File: cobra2snooty_test.go (hunk 1 above)
Comment (gssbzn) [reply cobra2snooty_test.go]: wanted to keep it simple but why not

**Pull request merged**